### PR TITLE
Document the way to file pull requests

### PR DESCRIPTION
**Pull request by Zlopez**
There is now possibility to file pull requests thanks to the Packit Team and the-new-hotness. Let's document this in the-new-hotness documentation, so users know about it.

### CODE REVIEW

File: docs/user-guide.rst [hunk 1]
```diff
@@ -60,3 +60,15 @@ For The New Hotness to function properly there have to be a project in
 `Anitya <https://release-monitoring.org>`_. For the information how to create
 a new project, look at the
 `user guide in Anitya documentation <https://anitya.readthedocs.io/en/stable/user-guide.html>`_.
+
+Filling pull requests on dist-git
```
Comment (nforro) [inline docs/user-guide.rst:64]: The right word is `filing`, but I think `opening` is a better fit here.

```suggestion
Opening pull requests in dist-git
```

File: docs/user-guide.rst [hunk 2]
```diff
@@ -60,3 +60,15 @@ For The New Hotness to function properly there have to be a project in
...
+---------------------------------
+
+Thanks to the work of `Packit <https://packit.dev/>`_ Team it is now possible to have
+pull requests filled directly in `dist git <https://src.fedoraproject.org>`_ when new
```
Comment (nforro) [inline docs/user-guide.rst:68]:
```suggestion
pull requests filed directly in `dist git <https://src.fedoraproject.org>`_ when a new
```

File: docs/user-guide.rst [hunk 3]
```diff
@@ -60,3 +60,15 @@ For The New Hotness to function properly there have to be a project in
...
+update is detected by The New Hotness for monitored project.
```
Comment (nforro) [inline docs/user-guide.rst:69]:
```suggestion
update is detected by The New Hotness for a monitored project.
```

File: docs/user-guide.rst [hunk 4]
```diff
@@ -60,3 +60,15 @@ For The New Hotness to function properly there have to be a project in
...
+
+The process is described in the
```
Comment (nforro) [inline docs/user-guide.rst:71]:
```suggestion
The process is described in
```

**Comment by Zlopez** [comment]
@nforro Thanks for the review, all your suggestion should be now added, but I first need to fix the tests.

**Review by nforro — APPROVED** [review]
:+1:

**Review by lachmanfrantisek — APPROVED** [review]
:+1: Thanks!

**Review by TomasTomecek — APPROVED** [review]
Perfect, thank you!

**Pull request merged**